src/server/debug2.rs

```diff
@@ -442,6 +467,30 @@ impl<ER: RaftEngine> Debugger for DebuggerImplV2<ER> {
     fn set_raft_statistics(&mut self, s: Option<Arc<RocksStatistics>>) {
         self.raft_statistics = s;
     }
+
+    fn get_range_properties(&self, start: &[u8], end: &[u8]) -> Result<Vec<(String, String)>> {
+        let mut props = vec![];
+        let regions = find_region_states_by_key_range(&self.raft_engine, start, end);
+        for (region_id, start_key, end_key, region_state) in regions {
+            let mut tablet_cache =
+                get_tablet_cache(&self.tablet_reg, region_id, Some(region_state)).unwrap();
+            let talbet = tablet_cache.latest().unwrap();
+            let mut prop = dump_write_cf_properties(
+                talbet,
+                start_key.as_ref().map(|k| k.as_bytes()).unwrap_or(start),
```

Comment:
It seems this should be data_key format.

Reply:
good catch, fixed